docs/html/reference/pip_install.rst

```diff
@@ -158,11 +158,16 @@ The following options are supported:
 
 .. pip-requirements-file-options-ref-list::
 
+Please note that the above options are global options, and should be specified on their individual lines.
+The options which can be applied to individual requirements are
+:ref:`--install-option <install_--install-option>`, :ref:`--global-option <install_--global-option>` and ``--hash``.
+
 For example, to specify :ref:`--no-index <install_--no-index>` and two
 :ref:`--find-links <install_--find-links>` locations:
 
 ::
 
+--pre
```

Comment:
This no longer matches the "For example" text above.

Reply:
So should I remove this line, or update the "For example" text above to include `--pre` ?

Reply:
I'd have removed the `--pre`, because the example is otherwise about setting some install locations, but it doesn't really matter either way, so I'm fine with how you changed it.